terraform/layers/app/iam.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_iam_role" "ec2_get_webfiles_role" {
+  name = "ec2-get-webfiles-role"
```

Comment:
this role will be visible within the console, so we need to apply the naming convention to show what project the role is for